src/Database/Query/CacheableQueryBuilder.php

```diff
@@ -81,8 +86,16 @@ public function __construct(
         $this->modelIdentifier = $cacheableProperties['identifier'] ?? null;
         $this->ttl = $cacheableProperties['ttl'] ?? null;
         $this->prefix = $cacheableProperties['prefix'] ?? null;
+        $this->logChannel = Arr::get($cacheableProperties, 'logging.channel', null);
         $this->logEnabled = Arr::get($cacheableProperties, 'logging.enabled', false);
-        $this->logLevel = Arr::get($cacheableProperties, 'logging.level', null);
+        $this->logLevel = Arr::get($cacheableProperties, 'logging.level', 'debug');
```

Comment:
Hey @Sleepy4k 

Why did you set the level to debug?
Please keep it as before, also below and lets just keep this PR focussed on the channel.

Thanks

Reply:
hei, about this, just want to optimize your current logging system which checking logChannel value for each logging, i thought this is gonna be a resolver for that as this issue still belong on logging tag, you sure want to set it on null?

Reply:
Hi, I might rethink this and you can also open another PR about it, it might make sense as it is quite some time back when I implemented it.
But I like to keep PRs atomic to a topic to not mix up changes and be picking later on. So, the case is not here if it makes sense or not, just that it should be a different topic for another day ^^

Reply:
aight i will revert this to before i touch it, thanks for it, i like your mindset tho